src/www/app/config.ts

```diff
@@ -19,4 +19,5 @@ export interface ShadowsocksConfig {
   password?: string;
   method?: string;
   name?: string;
+  extra?: {[key: string]: string};
```

Comment:
Let's remove this. See my comments on the storage format.

Reply:
Done in bc68e17. We now preserve the original access key and inject it into `OutlineServer`.